Jenkinsfile_CNP

```diff
@@ -70,10 +70,6 @@ withPipeline(type, product, component) {
     steps.archiveArtifacts allowEmptyArchive: true, artifacts: '**/reports/tests/**/*'
   }
 
-  onPR() {
```

Comment:
Removing enableCleanupOfHelmReleaseOnSuccess due to receiving this warning from Jenkins
<img width="500" alt="image" src="https://github.com/hmcts/wa-case-event-handler/assets/133241942/7bde7d4c-ae31-4d77-b696-faf5e9815447">

Reply:
sounds good